docs/ai/prompt-engineering-in-dot-net.md

```diff
@@ -0,0 +1,179 @@
+---
+title: "Prompt engineering in .NET"
+description: "Learn basic GPT prompt engineering concepts for .NET."
+author: catbutler
+ms.author: [your Microsoft alias or a team alias]
+ms.service: [the approved service name]
+ms.topic: concept-article #Don't change.
+ms.date: [mm/dd/yyyy]
+
+#customer intent: As a .NET developer, I want to understand GPT prompt engineering so that I can build more efficient and effective AI chat apps.
+
+---
+
+# Concept - Prompt engineering in .NET
+
+This article covers basic GPT prompt engineering for .NET, including Semantic Kernel and Azure OpenAI. 
+
+GPT models from OpenAI are prompt-based: they respond to user input text (a prompt) with the most likely series of words to follow (a completion). The basic completion in earlier GPT models is text generated in response to prompts. Newer GPT models produce completions in chat format. 
+
+Consider this text generation example:
+ 
+```Prompt
+"The president who served the shortest term was "
+```
+
+```Completion
+"Pedro Lascurain." 
+```
+
+Looks right, but what if your app is supposed to help US History students? Pedro Lascurain's 45-minute term is the shortest term for any president, but he served Mexico&mdash;the students are probably looking for "William Henry Harrsion." Clearly, the app could be more helpful to its intended users if you gave it some context. 
+
+That's the basic idea of prompt engineering: you add context to the prompt to help the model produce better completions. You can do this by giving the model instructions and examples.
+
+## Instructions
+
+This section explains the use of instructions in prompt engineering. 
+
+An instruction is text that tells the model how to respond. An instruction can be a directive (**You are helping students learn about US history, so talk about the US unless they specifically ask about other countries.**) or an imperative (**Translate to Tagalog:**). 
+
+Directives are more flexible than imperatives. 
+- A directive can provide more context, and you can combine several directives in one instruction. 
+- It's easier to implement a series of steps using a sequence of directives. You can spell out the steps that you want the model to take, but you can also just tell the model to break it into steps and then follow them, an approach called [chain of thought prompting](chain-of-thought-prompting.md).
```

Comment:
Yes it is. There might be times when you want to specify the steps, but letting the model do it will save time if you aren't concerned with the exact task breakdown. I'll revise to clarify, thanks!